packages/ui/style/tailwind.config.js

```diff
@@ -3,6 +3,7 @@ module.exports = function (app, options) {
     content: [
       `../../apps/${app}/src/index.html`,
       `../../apps/${app}/src/**/*.{ts,tsx,html,stories.tsx}`,
+      `../../apps/web/app/**/*.{ts,tsx,html,stories.tsx}`, // no src dir in web
```

Comment:
I see why you did this but really, we should be using a src directory in  apps/web since you wouldn't want to configure tailwind to be looking in the dist directory once the app has been built. That is why the original configuration was pointing to the src directory rather than the entire app directory. Tailwindcss will use this array of file paths to determine which files to scan through and compile into CSS. Keeping this array narrow in scope ensures that the speed of that compilation process is not compromised. I'm going to approve the commit even with this in there but I'll be removing it eventually once we have everything moved into an src directory in our next app.

Reply:
Understood. I didn't want to create a src directory and move the whole project/app in this commit. However, on that note, I made sure the path is narrowed enough to not have unnecessary files scanned. The "web/app" directory should technically only have the necessary components/pages for tailwind processing. I believe I actually narrowed it one level further. 

I just looked at a different project that has a 'src' directory, it sits on top of  'app', I could be wrong based on how this project is being configured.

Reply:
The way we have it now, we are missing the components directory for tailwind compilation. Eventually, with a src directory, it will contain both app and components and that line can be removed since existing configuration already accounts for that directory.